config/mistserver.dev.conf

```diff
@@ -1,223 +0,0 @@
-        "version": false
-      },
-      {
-        "connector": "livepeer-analyzer",
-        "j": false,
-        "port": 8008,
-        "prometheus": true
-      },
-      {
-        "broadcasters": "[{\"address\":\"http://127.0.0.1:8935\"}]",
-        "connector": "livepeer-api",
-        "fallback-proxy": "http://localhost:3012",
-        "json": false,
-        "jwt-audience": "localhost",
-        "jwt-secret": "F696D626-75C5-42D7-97E1-678E078532FA",
-        "require-email-verification": false,
-        "return-region-in-orchestrator": false,
-        "supress-record-in-hook": false
-      },
-      {
-        "api-server": "http://localhost:3004",
-        "api-token": "f5914701-fc12-4b3d-b203-b99951e07481",
-        "connector": "livepeer-mist-api-connector",
-        "j": false,
-        "mist-creds": "test:test"
-      },
-      {
-        "connector": "livepeer-www"
-      },
-      {
-        "connector": "livepeer-catalyst-api",
-        "port": "7979"
-      },
-      {
-        "connector": "livepeer-task-runner",
-        "j": false
-      },
-      {
-        "autoAdjustPrice": false,
-        "broadcaster": false,
-        "cliAddr": "127.0.0.1:7936",
-        "connector": "livepeer",
-        "currentManifest": false,
-        "httpIngest": false,
-        "initializeRound": false,
-        "j": false,
-        "localVerify": false,
-        "metricsClientIP": true,
-        "metricsPerStream": true,
-        "monitor": true,
-        "orchestrator": true,
-        "redeemer": false,
-        "reward": false,
-        "serviceAddr": "127.0.0.1:8936",
-        "testTranscoder": false,
-        "transcoder": true,
-        "version": false
-      },
-      {
-        "connector": "livepeer-victoria-metrics",
-        "j": false,
-        "promscrape.config": "./config/scrape_config.yaml"
-      },{
-        "connector": "livepeer-vmagent",
-        "j": false,
-        "promscrape.config": "./config/scrape_config.yaml",
-        "remoteWrite.url": "http://localhost:8481",
-        "remoteWrite.label": "region=local-dev",
-        # "remoteWrite.basicAuth.username": "username",
-        # "remoteWrite.basicAuth.password": "password"
-      }
-    ],
-    "serverid": null,
-    "triggers": {
-      "CONN_CLOSE": [
-        {
-          "handler": "http://localhost:7933/",
-          "sync": false
-        }
-      ],
-      "LIVE_TRACK_LIST": [
-        {
-          "handler": "http://localhost:7933/",
-          "sync": false
-        }
-      ],
-      "PUSH_END": [
-        {
-          "handler": "http://localhost:7933/",
-          "sync": false
-        }
-      ],
-      "PUSH_OUT_START": [
-        {
-          "handler": "http://localhost:7933/",
-          "sync": false
-        }
-      ],
-      "PUSH_REWRITE": [
-        {
-          "default": "000reallylongnonexistenstreamnamethatreallyshouldntexist000",
-          "handler": "http://localhost:7933/",
-          "sync": true
-        }
-      ]
```

Comment:
Why did we removed triggers from conf?

Reply:
These are the `mist-api-connector` triggers that connect it to Studio. While we still want to retain the ability to stand up a full Studio development environment when necessary, it's overkill for most Catalyst development.